src/ClusterToggleView/ClusterToggleView.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ClusterThumbView from '../ClusterThumbView';
+import ClusterDetailView from '../ClusterDetailView';
+
+//Default state
+//mode : Thumbnail | Detail
+
+let THUMBNAIL_MODE = "thumbnail";
```

Comment:
I think this is a great use case for the `const` keyword.  variables declared with `const` cannot be mutated or reassigned.  We should probably use `const` by default and only use `let` when know we want to be able to change it.  Something to discuss next meeting.